Gemfile

```diff
@@ -45,11 +45,7 @@ gem 'pg_search'
 gem 'puma', '~> 5.6.4'
 gem 'pundit', '~> 2.2.0'
 gem 'rack-cors', '~> 1.0', require: 'rack/cors'
-if next?
```

Comment:
don't remove the `next?` branch - we'll need this to keep the application dual bootable for the next upgrade. Instead just update the gem versions inside the brances, e.g. 
```
if next?
 gem 'rails', '~> 5.0'
else
 gem 'rails', '~> 5.0'
end